tornimo-spring-aws/src/main/java/io/tornimo/cloud/aws/AwsEc2Metadata.java

```diff
@@ -0,0 +1,56 @@
+package io.tornimo.cloud.aws;
+
+import com.amazonaws.util.EC2MetadataUtils;
+
+public class AwsEc2Metadata {
+
+    private final String region;
+    private final String availabilityZone;
+    private final String instanceType;
+    private final String instanceId;
+    private final String account;
+
+    public static AwsEc2Metadata create() {
```

Comment:
Maybe it would be more consistent to have it in `AwsEc2Endpoint` like for ecs?

Reply:
It would be nicer to have it in a separated class but this is easier for me to test.